CalibTracker/SiStripChannelGain/plugins/SiStripGainFromData.cc

```diff
@@ -888,8 +888,7 @@ void SiStripGainFromData::algoEndJob() {
 
     unsigned int GOOD = 0;
     unsigned int BAD = 0;
-    double MPVmean = MPVs->GetMean();
-    MPVmean = 300;
+    double MPVmean = 300.;  //MPVs->GetMean();
```

Comment:
Should this really be hardcoded? Does having the `MPVs->GetMean()` alone lead to convergence issues?

Reply:
That's what was before. I am not changing the behavior, just removing the SA warning.

Reply:
Right, I guess I could speak to the code authors, I was just surprised to see this magic number here

Reply:
that's not a magic number, it's the target MVP post-calibration.